kvredis/src/main.rs

```diff
@@ -73,18 +62,24 @@ impl ProviderHandler for KvRedisProvider {
         };
 
         if let Ok(client) = redis::Client::open(redis_url) {
-            if let Ok(connection) = client.get_async_connection().await {
-                let mut update_map = self.actors.write().await;
-                update_map.insert(
-                    ld.actor_id.to_string(),
-                    RwLock::new(RedisConnection::new(connection, redis_url)),
-                );
-            } else {
-                warn!(
-                    "Could not create Redis connection for actor {}, keyvalue operations will fail",
-                    ld.actor_id
-                )
-            }
+            // if let Ok(connection) = client.get_async_connection().await {
+            let mut update_map = self.actors.write().await;
+
+            update_map.insert(
+                ld.actor_id.to_string(),
+                RwLock::new(
+                    client
+                        .get_tokio_connection_manager()
+                        .await
+                        .map_err(to_rpc_err)?,
```

Comment:
How friendly are these error messages? Should we prefix them with a friendly string? Also, I thought we didn't want to return RpcErrors in `put_link` anymore

Reply:
great catch, fixed